test/test_helpers.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+# SPDX-FileCopyrightText:  PyPSA-Earth and PyPSA-Eur Authors
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# -*- coding: utf-8 -*-
+
+import pathlib
+from test.conftest import _content_temp_file, get_temp_file
+
+from scripts._helpers import (
+    get_abs_path,
+    get_basename_abs_path,
+    get_dirname_abs_path,
+    get_dirname_path,
+    get_path,
+    get_path_size,
+)
+
+path_cwd = str(pathlib.Path.cwd())
+
+
+def test_get_abs_path():
```

Comment:
A nice test would be to check if the `os` thing you are replacing is equal to the pathlib solution

Reply:
done! I have added extra assert statements in test/test_helpers.py :D